src/Scrolling.md

```diff
@@ -4,11 +4,19 @@
 These registers can be accessed even during Mode 3, but modifications may not take
 effect immediately (see further below).
 
-## FF42–FF43 — SCY, SCX: Viewport Y position, X position
+## FF42–FF43 — SCY, SCX: Background viewport Y position, X position
```

Comment:
Indeed, I agree this name is better!
Note, however, that changing the section's name created [some broken links](https://github.com/gbdev/pandocs/pull/506/checks#step:7:211), which must be updated as well.